src/main/java/org/webonise/listapp/DbConnection.java

```diff
@@ -0,0 +1,30 @@
+package org.webonise.listapp;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+
+public class DbConnection {
+    private static DbConnection dbconnection = new DbConnection();
+    private DbConnection(){
+        // This is a Private Constructor to maintain only one instance.
+    }
+    private Connection createConnection() {
+        Connection connection = null;
+        try {
+            Class.forName("org.postgresql.Driver");
+            connection = DriverManager.getConnection("jdbc:postgresql://127.0.0.1:5432/mydatabase","webonise","12345678");
+        } catch (Exception e) {
+            System.out.println("Connection Failed.Check console");
+            e.printStackTrace();
+            return null;
```

Comment:
don't return null, if connection could not established, default null will be returned as you are returning connection. Also, returning null from catch will not execute anything further, you're printing something after that. Check it please.